ui/admin/tests/unit/controllers/scopes/scope/workers/worker/create-tags-test.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module(
+  'Unit | Controller | scopes/scope/workers/worker/create-tags',
+  function (hooks) {
+    setupTest(hooks);
+
+    test('it exists', function (assert) {
+      let controller = this.owner.lookup(
+        'controller:scopes/scope/workers/worker/create-tags',
+      );
+      assert.ok(controller);
+    });
+  },
+);
```

Comment:
i think the test cases for the actions are missing here, only if possible to test.

Reply:
Thanks. Added.